src/utils/addresses.ts

```diff
@@ -0,0 +1,35 @@
+import { getAddress } from '@ethersproject/address'
```

Comment:
this is unchanged from existing besides naming update to `shortenAddressIfValid` to avoid collision.